package.json

```
@@ -5,7 +5,7 @@
   "main": "src/node.js",
   "scripts": {
     "lint": "./node_modules/.bin/standard --verbose src/**",
-    "test": "rm -rf ./tmp ; jest --forceExit --detectOpenHandles --coverage --runInBand",
+    "test": "rm -rf ./tmp ; rm -rf ./test ; mkdir -p ./test/one ; jest --forceExit --detectOpenHandles --coverage --runInBand",
```

What is the `./test` directory? I don't see it getting used anywhere?

---

Just creating a test directory to make the test pass. May be overcomplicated

---

I'm not sure I follow.

---

It creates one directory on the fly for the `it counts the number of directories` test. After that, the directory is deleted

---

oh, I see! Can you put this into the `tmp` dir? will make things simpler. Something like this
```suggestion
    "test": "rm -rf ./tmp ; mkdir -p ./tmp/test/one ; jest --forceExit --detectOpenHandles --coverage --runInBand",
```